root/components/common-macros.tt

```diff
@@ -931,7 +931,7 @@ note-[%- edit.id -%]-[%- loop.count() -%]
     (description == '' ? '' : '&description=' _ uri_escape(description));
 END ~%]
 
-[%- MACRO remove_from_merge_header(to_merge) BLOCK -%] [% # Converted to React at root/components/RemoveFromMergeTableHeader.js %]
+[%- MACRO remove_from_merge_header(to_merge) BLOCK -%] [% # Converted to React as part of entity lists %]
```

Comment:
In two months “entity lists” might not be telling anymore, I suggest to keep referring to code.

```suggestion
[%- MACRO remove_from_merge_header(to_merge) BLOCK -%] [% # Converted to React at root/utility/tableColumns.js %]
```

Reply:
Resolved in https://github.com/metabrainz/musicbrainz-server/pull/1428/commits/6e40d6b97042e4f9d8a686c508c98c6bf027245a